bin/ch/WScriptJsrt.cpp

```diff
@@ -1490,7 +1527,22 @@ HRESULT WScriptJsrt::ModuleMessage::Call(LPCSTR fileName)
         errorCode = specifierStr.GetError();
         if (errorCode == JsNoError)
         {
+            char oldpath[_MAX_PATH];
+            bool useReferenceModulePath = false;
+            if (this->moduleRecord)
+            {
+                auto moduleDirEntry = moduleDirMap.find(this->moduleRecord);
+                if (moduleDirEntry != moduleDirMap.end())
+                {
+                    _getcwd(oldpath, _MAX_PATH);
+                    useReferenceModulePath = true;
+                    std::string dir = moduleDirEntry->second;
+                    _chdir(dir.c_str());
```

Comment:
Changing current directory in an multi-thread application is usually frown upon. , because is the property is process wide.